docs/developer/style.md

```diff
@@ -2,18 +2,328 @@
+| ERROR     | The message that indicates the application is having a serious issue or,<br>represent the failure of some important going on in the application.<br>It does not cause the application to go down.<br>Someone must investigate the error later. | Failed to insert an entry into the database, with retry count exceeded.<br>Failed to update the cache, and the cache is not important.             | User 1 is failed to insert in the database, errors:<br>retry count: 1, error: ErrMsg1, retry count: 2, error: ErrMsg2, ....                                                                                            |
+| FATAL     | Message that indicate the application is corrupting or having serious issue.<br>The application will go down after logging the fatal error. <br>Someone must investigate and resolve the fatal as soon as possible.                            | Failed to init the required cache during the application start.                                                                                    |                                                                                                                                                                                                                        |
+
+## Program comments
+
+Program comments make easier to understand the source code. We suggest not to write many comments inside the source code unless the source code is very complicated and confusing; otherwise we should divide the source code into methods to keep the readability and usability of the source code.
+
+Everyone should write the comments to all the public objects on your source code, like public packages, interface, structs, methods, and even public constant and variable. The godoc will be generated base on the comment of source code.
+
+## Documentation
+
+Documentation is generated from the program comments. Please refer to [Godoc](https://godoc.org/github.com/vdaas/vald) for the program documentation.
+
+## Internal packages
+
+Vald implements its internal package to extend and customize the functionality of the standard library and third-party library.
+We should use the internal package instead of standard libray to implement Vald.
+Please refer to [godoc](https://godoc.org/github.com/vdaas/vald/internal) for the internal package document.
+
+## Dependency management and Build
+
+We should use `go mod tidy` to manage the `go.mod` file in the project.
+
 ## Style
 
 ## Test
 
-Testing guideline has 2 important rules for the coding quality and readability
+The testing guideline has 2 important rules for the coding quality and readability
```

Comment:
```suggestion
The testing guideline has 3 important rules for the coding quality and readability:
```

Reply:
How is it? @kevindiu

Reply:
I fixed in this commit.
https://github.com/vdaas/vald/pull/320/commits/3a0fe532d21383cbe7e835e7652338d8a2d11540